### ISSUE TITLE
How to account for missing values in an array?

### ISSUE THREAD

**Issue by emmansh** [FabrizioSandri/r-integration#9]
R marks missing values by denoting `NA`. The equivalent in JS in `NaN`. Should `r-integration` convert `NA` to `NaN` (and vice versa)?

Example
---
```javascript
// in node js
const R = require('r-integration');
const arrayOfStrings = ["a", "b", "c", NaN, "f", "g"];
let result = R.callStandardMethod("toupper", arrayOfStrings);
console.log(result)
```  
The output:
> [
>   'A', '',    '',  'B',
>   '',  '',    'C', '',
>   '',  'NAN', 'F', '',
>   '',  'G',   '',  ''
> ]

Aside from the blanks that are already discussed in #8, please note that `NaN` was treated in R as a string, and thus was converted to uppercase as the other array elements. Instead, what I would _expect_ is that the `NaN` element will remain `NaN` when returned from R and stored to `result`.

**Comment by FabrizioSandri** [comment]
The space problem is resolved as mentioned in #8 . 

The NaN problem seems to be a R problem. From the documentation:
> NaN (“Not a Number”) means 0/0

and if applied to the `toupper` function there is an implicit conversion to string. This is a strange behavior I know. 
If you try from a R console to execute the following you will see the result:
```R
> toupper(NaN)
[1] "NAN"
```

This is something independent from the `r-integration` library

**Comment by emmansh** [comment]
I see. And what if we had this JS array:
```javascript
// in node js console
> ["a", "b", "c", , "f", "g"]
[ 'a', 'b', 'c', <1 empty item>, 'f', 'g' ]
```

```javascript
// in my node js script
const R = require('r-integration');

const arrayOfStrings = ["a", "b", "c", , "f", "g"]
let result = R.callStandardMethod("toupper", arrayOfStrings);
console.log(result)
```
Output to console is:
> [ 'A', 'B', 'C', 'F', 'G' ]
But I would expect:
> [ 'A', 'B', 'C', <1 empty items>, 'F', 'G' ]

---
The equivalent in R is:
``` r
my_str <- c("a", "b", "c", NA, "e", "f")
toupper(my_str)
#> [1] "A" "B" "C" NA  "E" "F"
```

**Comment by FabrizioSandri** [comment]
Solved with the new release, version `2.2.0` which supports NaN and NA values.
